Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml

```diff
@@ -0,0 +1,777 @@
+      type: String
+    - contextPath: TrendMicroCAS.EmailSweep.value.mailbox
+      description: Mailbox where the email message is
+      type: String
+    - contextPath: TrendMicroCAS.EmailSweep.value.source_domain
+      description: Source domain of the email message
+      type: String
+    - contextPath: TrendMicroCAS.EmailSweep.value.source_ip
+      description: Source IP address of the email message
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: |-
+        ACCOUNT_DISABLE: disables a user account
+        ACCOUNT_ENABLE_MFA: enforces a user to perform a multi-factor authentication before being forced to change their password
+        ACCOUNT_RESET_PASSWORD: requests to reset the password for a user account
+      isArray: false
+      name: action_type
+      predefined:
+      - ACCOUNT_DISABLE
+      - ACCOUNT_ENABLE_MFA
+      - ACCOUNT_RESET_PASSWORD
+      required: true
+      secret: false
+    - default: false
+      description: Email addresses, support comma separated string
+      isArray: false
+      name: account_user_email
+      required: true
+      secret: false
+    deprecated: false
+    description: Takes actions on a batch of specified user accounts, including disabling
+      a user account, requesting to enable multi-factor authentication (MFA) for a
+      user account, and requesting to reset password for a user account.
+    execution: false
+    name: trendmicro-cas-user-take-action
+    outputs:
+    - contextPath: TrendMicroCAS.UserTakeAction.action_type
+      description: the action type.
+      type: String
+    - contextPath: TrendMicroCAS.UserTakeAction.account_user_email
+      description: list of account user email.
+      type: String
+    - contextPath: TrendMicroCAS.UserTakeAction.batch_id
+      description: Unique ID of the API request, including all actions to take on
+        user accounts specified within this request.
+      type: String
+    - contextPath: TrendMicroCAS.UserTakeAction.traceId
+      description: Randomly generated identity to uniquely trace the request.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: |-
+        Action to take on an email message. Options include:
+        MAIL_DELETE: deletes an email message
+        MAIL_QUARANTINE: quarantines an email message
+      isArray: false
+      name: action_type
+      predefined:
+      - MAIL_DELETE
+      - MAIL_QUARANTINE
+      required: true
+      secret: false
+    - default: false
+      description: Email address of an email message to take action on
+      isArray: false
+      name: mailbox
+      required: true
+      secret: false
+    - default: false
+      description: |-
+        Internet message ID of an email message to take action on
+        It can be obtained from "trendmicro-cas-email-sweep" command
+      isArray: false
+      name: mail_messge_id
+      required: true
```

Comment:
check if both `mail_message_id` and `mail_unique_id` are required

Reply:
both  are required